config/initializers/hosting_environment.rb

```diff
@@ -0,0 +1 @@
+require "#{Rails.root}/lib/hosting_environment.rb"
```

Comment:
I think you don't need to add this if you put `HostingEnvironment` in `/app/lib` instead of `/lib`